File: app/src/main/java/cn/readsense/com/calendardemo/service/BindActivityCallBack.java
package cn.readsense.com.calendardemo.service;

import java.util.List;

/**
 * @description:
 * @author: dongyiming
 * @date: 2018/9/28 22:44
 * @version: v1.0
 */
public interface BindActivityCallBack<T> {

    /**
     * <TODO>
     *
     * @param:ref：0为左边时间
     * @author: dongyiming
     * @date: 2018/9/25 0:13
     * @version: v1.0
     */
    void call(List<T> list, int ref);
}

File: app/src/main/java/cn/readsense/com/calendardemo/BindActivityCallBack.kt
package cn.readsense.com.calendardemo


/**
 * @description: <fragment->activity>
 * @author: dongyiming
 * @date: 2018/9/24 23:44
 * @version: v1.0
 */
interface BindActivityCallBack<in T> {

    /**
     * <TODO>
     * @param:ref：0为左边时间
     * @author: dongyiming
     * @date: 2018/9/25 0:13
     * @version: v1.0
     */
    fun call(list: List<T>, ref: Int)
}